### ISSUE TITLE
Change speaker color for those who are color anomaly

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#1754]
For those who are color anormaly the combination of red and blue is uncomfortable, like this case on Youtube.

![](http://i.imgur.com/bqnv8bS.jpg)

I propose to change blue into white, same as the title color.

**Comment by luixxiul** [comment]
Favicon into the white, too. It just looks like a triangle.

**Comment by luixxiul** [comment]
Ah, in this case I just have to disable "Show tabs in page theme color"..

**Comment by diracdeltas** [comment]
More generally we should make sure we are using accessible colors in the browser UI. When I worked at Yahoo, we were given color pallette sets so that developers didn't just randomly pick colors every time they had to make a button, etc. Maybe @bradleyrichter can help with making a `less/colors.less` file for defining common Brave colors?

It looks like there's various tools like http://accessibility.umn.edu/color-and-contrast-414.html for determining whether colors are accessible.

**Comment by bradleyrichter** [comment]
I wonder if we can add something to prevent this problem where favicons disappear into the BG color of the tab. Since we originally took a favicon color sample, we could take that and DIF the HSV between itself and the tab theme color. Then we can either invert on a threshold value or apply a CSS filter to greyscale the icon.

(just ideas...)

We may need to shift away from blue on the audio indicator regardless though....

And eventually create a color table...

**Comment by bbondy** [comment]
We do the color comparison for picking black or white text already. If we used that as the color it would simplify this a lot.

**Comment by lirenchan** [comment]
I wish that High Contrast #2 (green text on black background) in Windows 10 will soon work in Brave for Windows; Edge browser works the best for me with High Contrast #2, thus it is still my default browser for Windows 10.  The inverted video mode on Mac already works in Brave for Mac, many thanks.

**Comment by luixxiul** [comment]
Setting the same speaker color as the tab title might be a solution:

![clipboard01](https://user-images.githubusercontent.com/3362943/32545898-739466be-c4c0-11e7-9e41-f52d4a053d84.png)

**Comment by bsclifton** [comment]
Closing in favor of https://github.com/brave/brave-browser/issues/906
